extensions/wikia/AdEngine/AdEngine2ContextService.class.php

```diff
@@ -52,8 +52,9 @@ public function getContext( Title $title, $skinName ) {
 					'showAds' => $adPageTypeService->areAdsShowableOnPage(),
 					'trackSlotState' => $wg->AdDriverTrackState,
 					'usePostScribe' => $wg->Request->getBool( 'usepostscribe', false ),
-					'sourcePointUrl' => $sourcePointUrl,
+					'sourcePointUrl' => $sourcePointRecoveryUrl,
```

Comment:
That's right. The same story as `InstantGlobals.wgAdDriverSourcePointCountries`.